src/Engine/GraphicsHandler/I_GraphicsHandler.h

```diff
@@ -8,19 +8,191 @@
 class I_GraphicsHandler {
 public:
     virtual ~I_GraphicsHandler() {}
+    /**
+     * Used to intiate the GraphicsHandler
+     * Loads the vector of <code>graphic_paths</code> which are file paths
+     * to <code>GraphicsHandler::game_graphics_</code>, which is a map of std::string to SDL_Textures*
+     * @param path is the relative (recommended) or absolute path to graphics assets that will 
```

Comment:
why is relative recommended? 

I think the format of the comment should refer to the name of the actual arg so

```
graphics_paths is a vector of relative (recommended) or absolute paths...
```

Reply:
interesting. I thought it was path for some reason. 

relative because if you specify absolute path, my path to assets is: `/home/byron/Documents/TerminalFighter/assets`, while yours could be completely different.